contingency/contingency-api/src/test/java/com/powsybl/contingency/tasks/HvdcLineTrippingTest.java

```diff
@@ -65,7 +65,7 @@ public void lineTrippingTest() {
         contingency.toTask().modify(network, null);
 
         assertTrue(terminal1.isConnected());
-        assertFalse(terminal2.isConnected());
+        assertTrue(terminal2.isConnected());
```

Comment:
I think this is a regression, so I have to check it: if you trigger an AC line, both side are disconnected. I used the same approach for HVDC: I disconnected the 2 converter stations, so both terminal should be disconnected.

Reply:
Thanks @mathbagu Indeed you are right we have an issue here because the HVDC converter station is considered as a feeder and as a branch.